src/huggingface_hub/file_download.py

```diff
@@ -495,53 +490,90 @@ def http_get(
     if len(displayed_name) > 40:
         displayed_name = f"(…){displayed_name[-40:]}"
 
+    consistency_error_message = (
+        f"Consistency check failed: file should be of size {expected_size} but has size"
+        f" {{actual_size}} ({displayed_name}).\nWe are sorry for the inconvenience. Please retry download and"
+        " pass `force_download=True, resume_download=False` as argument.\nIf the issue persists, please let us"
+        " know by opening an issue on https://github.com/huggingface/huggingface_hub."
+    )
+
     # Stream file to buffer
-    progress = tqdm(
+    with tqdm(
         unit="B",
         unit_scale=True,
         total=total,
         initial=resume_size,
         desc=displayed_name,
         disable=bool(logger.getEffectiveLevel() == logging.NOTSET),
-    )
-    try:
+    ) as progress:
+        if hf_transfer and total is not None and total > 10 * DOWNLOAD_CHUNK_SIZE:
```

Comment:
Arbitrarily setting this to 50MB. Thanks for the explanation here :) 

```suggestion
        if hf_transfer and total is not None and total > 5 * DOWNLOAD_CHUNK_SIZE:
```